tests/testit/test-base64.R

```diff
@@ -31,3 +31,23 @@ assert('base64_encode_r() returns the same result as base64_encode()', {
   f = R_logo()
   base64_encode_r(f) %==% base64_encode(f)
 })
+
+assert('base64_decode() does not accept a non-string input', {
+  has_error(base64_decode(x = 42))
```

Comment:
Let's always add `()` on test conditions, just in case we need to add more conditions here in `{}` in the future:

```suggestion
  (has_error(base64_decode(x = 42))
```